server/app/controllers/monitoring/MetricsController.java

```diff
@@ -21,11 +24,45 @@ public final class MetricsController extends CiviFormController {
 
   private final boolean metricsEnabled;
   private final CollectorRegistry collectorRegistry;
+  private final Database database;
+
+  private static final Counter queryMetricCount =
```

Comment:
**nitpick:** SCREAMING_SNAKE_CASE for constants

Reply:
Updated! As well as made the changes we discussed.